modules/logging-upgrading-clo.adoc

```diff
@@ -16,6 +16,39 @@ To update the {clo} to a new major release version, you must modify the update c
 
 .Procedure
 
+. Optional: When upgrading to the {clo} 5.9 or later, you must add the `v13` schema version to the `LokiStack` custom resource before the upgrade:
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+# ...
+spec:
+# ...
+  storage:
+    schemas: # <1>
+    - effectiveDate: "<yyyy>-<mm>-<past_dd>" # <2>
+      version: v12 # <3>
+    - effectiveDate: "<yyyy>-<mm>-<future_dd>" # <4>
+      version: v13 # <5>
+# ...
+----
+<1> Upgrading to the {clo} 5.9 or later without the `v13` schema version in the `LokiStack` custom resource will result in a web console warning.
+<2> Keep the previously set date value.
+<3> Later, after the `effectiveDate` date of the `v13` schema version, you can remove the `v12` schema version.
```

Comment:
I'd make it simplest leaving only in <2> something like:

<2> Don't delete the current schema definition or data won't be accesible.

Reply:
If we tell the users to keep `v12`, they will keep it, but will they need it in the LokiStack CR afterwards? Imagine I am a user, and I'm wondering why I should keep `v12` there.

Reply:
If an error is done when editing, the collectors won't be able to write new logs and an user  won't be able to see the logs stored in the permanent storage. Then, the impact is bigger and it could need longer explanation (this explanation is in the upstream) for recovering the visibility of the logs if the details in how it was before remains in some place.

Being present the old schema entry is not a problem per se, when not more logs with the current schema as deleted, then, those lines continue in the configuration, but they are not doing anything. 

I don't feel that "deleting" an old schema should be part of the upgrade activity where it should be maintained the simplest possible focusing in the upgrade steps. 

How to recover from a wrong deletion of the schema, in case that desired to document, it should be in a different section where also explained how to recover from that situation. Then, as said, let's try to get the update section clear, simple and if someone requires it, then, it should be documented out of the upgrade process, for example in "maintaining loki" or whatever other section for Loki.